lib/rmt.toit

```diff
@@ -58,16 +61,22 @@ class Signals:
   /** The number of signals in the collection. */
   size/int
 
-  bytes_/ByteArray
+  /**
+  The resolution of the signals in Hz.
 
-  /** The empty signal collection. */
-  static ZERO ::= Signals 0
+  Null, if this instance was created without a resolution.
```

Comment:
I think I'd reword this to avoid capitalizing `null`. 

```suggestion
  If this instance was created without a resolution, $resolution is null.
```